### PR TITLE
[Android][Accessibility] TextBlock style: heading

### LINKED ISSUE #5480: [Android] Renderer support for TextBlock headings

### PR DESCRIPTION
**Pull request by golddove**
## Related Issue
Fixes #5480 

## Description
`TextBlock` with `style: heading` gets the appropriate accessibility property.

## Sample Card
v1.5/Elements/TextBlock.Style

## How Verified
Recording is poor, but you can hear the audio:

https://user-images.githubusercontent.com/4442788/112674304-21babc00-8e3c-11eb-98ed-311b7c2c26b5.mp4

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/5587)

### CODE REVIEW

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/TextBlockRenderer.java [hunk 1]
```diff
@@ -69,6 +70,24 @@ public static void setTextSize(TextView textView, FontType type, TextSize textSi
...
+    /**
+     * Applies given TextStyle to the given TextView
+     * @param textView TextView to apply style to
+     * @param style TextStyle to apply
+     */
+    private static void applyTextStyle(TextView textView, final TextStyle style)
+    {
+        // Indicate Heading to accessibility service
+        // TODO: Refactor to ViewCompat.setAccessibilityHeading after AndroidX upgrade
+        ViewCompat.setAccessibilityDelegate(textView, new AccessibilityDelegateCompat() {
```
Comment (jwoo-msft) [inline source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/TextBlockRenderer.java:82]: neat

**Review by jwoo-msft — APPROVED** [review]

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/TextBlockRenderer.java (hunk 1 above)
Comment (jwoo-msft) [reply source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/TextBlockRenderer.java:82]: is it part of Android SDK? wish iOS has something similar, iOS VO seems to have a timing-related. bug

**Review by jwoo-msft — APPROVED** [review]

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/TextBlockRenderer.java (hunk 1 above)
Comment (paulcam206) [reply source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/TextBlockRenderer.java:82]: UWP has something similar

https://github.com/microsoft/AdaptiveCards/blob/2257e48ae9a795ea33760d442f0a6b1523c6c682/source/uwp/Renderer/lib/LinkButton.cpp#L8-L12

**Review by paulcam206 — APPROVED** [review]

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/TextBlockRenderer.java (hunk 1 above)
Comment (golddove) [reply source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/TextBlockRenderer.java:82]: Yeah, need to use the delegate on the version of support library we're using. Newer versions have a direct property similar to iOS.

**Pull request merged**